openslides_backend/action/actions/user/json_upload.py

```diff
@@ -122,6 +130,22 @@ def generate_entry(
                 else:
                     state = ImportState.NEW
                     entry["username"] = {"value": entry["username"], "info": "done"}
+            elif entry.get("saml_id"):
+                if self.check_saml_id_for_duplicate(entry["saml_id"], payload_index):
+                    state = ImportState.DONE
+                    if searchdata := self.get_search_data(payload_index):
+                        entry["saml_id"] = {
+                            "value": entry["saml_id"],
+                            "info": "done",
+                            "id": searchdata["id"],
+                        }
+                    else:
+                        entry["saml_id"] = {"value": entry["saml_id"], "info": "done"}
+                        state = ImportState.ERROR
+                        messages.append(f"Duplicate in csv list index: {payload_index}")
```

Comment:
Why is this a duplicate? Couldn't it be that the user just didn't exist yet?

Also you can unify this whole `saml_id` section with the `username` one above it.

Reply:
Unified.